CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 All notable changes included in each Chainflip release will be documented in this file.
 
-## [1.6.4] - 2024-09-06
+## [1.6.5] - 2024-09-10
```

Comment:
We should still have a 1.6.4 release note? we've already released it

Reply:
Yeah.. not sure what happened there.

Reply:
Ah it snuck into the version bump commit - need to be more careful with that...